general-concepts/slotting/text.xml

```diff
@@ -45,6 +52,102 @@ To <c>DEPEND</c> upon a package in a specific slot, refer to
 </p>
 </body>
 
+<section>
+<title>Considerations for slotting</title>
+<body>
+
+<p>
+Slotting is a powerful tool and as such, is easily abused. It's critical
+that when slotting, package maintainers carefully consider the impact
+on <e>reverse dependencies</e>.
+</p>
+
+<p>
+Do not be fooled by the ease in adapting the package itself to slotting:
+slotting the package itself is always the easiest part.
+</p>
+
+<p>
+Before proceeding at all, consider the impact on the wider ecosystem
```

Comment:
I really dislike that "ecosystem" metaphor in computing (especially in free-software) context. It is a fuzzy and ambiguous term.

Reply:
Can you suggest an alternative? In this case, I really want people to think "how is this going to affect other people", and "is there a thing this is going to break far beyond Gentoo"?

Reply:
"Before proceeding, consider the wider implications beyond Gentoo."